models/cex/algorand/cex_algorand_addresses.sql

```diff
@@ -0,0 +1,23 @@
+{{config(
+        tags = ['static'],
+        alias = 'addresses',
```

Comment:
can you add `schema` property to all of these new ones?

Reply:
done ☑️